src/pages/submit/grant.tsx

```diff
@@ -298,6 +303,13 @@ export default function SubmitGrant() {
           />
 
           <Container className="ContentLayout__Container">
+            {submissionVpNotMet && (
+              <ContentSection className="ProjectRequestSection__Content">
+                <Text className="GrantRequest__SubmissionVpNotMetText" size="lg" color="primary">
+                  {t('error.grant.submission_vp_not_met', { threshold: SUBMISSION_THRESHOLD_GRANT })}
+                </Text>
+              </ContentSection>
```

Comment:
Is the submission button disabled as well?

Reply:
`disabled={!allSectionsValid || isFormDisabled || submissionVpNotMet}` ?

Reply:
Good catch